### DISCUSSION TITLE
Is the Post Install script working correctly?

### DISCUSSION THREAD

**Discussion by antamy** [tteck/Proxmox#2725] [Q&A] [answered]
I have 2 Proxmox installations that are working fine, on both I applied the Proxmox VE Post Install helper script.  I am trying to spin up another installation, where I install from the same USB stick as the others (PM 8.1.1). I run the post install script and the PM update runs.  However, PM doesn't update!  It is "stuck" at v8.1.10, whereas the others are at v8.1.3.   If I look at Updates on the new host, it is empty.  On the other hosts, there are long lists under Origin: Debian and Origin: Proxmox.   If I look at Repositories, the new host is using deb.debian.org, whereas the other two use ftp.debian.org.   So, something has changed!

How can I get Proxmox to update with the script?

**Comment by tteck** [comment, marked as answer]
8.1.10 is the latest version, 8.1.3 is old.

**Reply by antamy** [reply]
Ah, I was reading it as 8.1.1, not .10!  My bad!
